src/lib.rs

```diff
@@ -40,8 +41,24 @@ pub mod convert;
 #[cfg(feature = "std")]
 use std::vec::Vec;
 
-pub use crate::types::*;
 use crate::utils::*;
+use types::*;
+
+/// Re-export of all the basic features.
+pub mod prelude {
+    pub use crate::*;
+
+    pub use crate::types::*;
```

Comment:
My only complaint is this may conflict with other crates, but I see it from the documentation PR that the prelude is optional.

Reply:
Basically that means this PR fixes the problem introduced by #57.

Reply:
Yes. Plus `types` conflicting with other crates is not a problem here, as what is brought into scope are the items within `types` rather than `types` itself.

Reply:
`pub use crate::types::*;` above was pulling in the types from the module, that's what I've meant.